src/config-schema.ts

```diff
@@ -25,6 +25,24 @@ export const configSchema = {
     _type: Type.String,
     _default: "Pharmacy",
   },
+  actionButtons: {
```

Comment:
added some config parameters so that those who don't want to use the "Pause" and "Close" buttons can disable them